models/Document/Dao.php

```diff
@@ -322,10 +322,16 @@ public function deleteAllPermissions()
 
     /**
      * Deletes all scheduled tasks assigned to the document.
+     *
+     * @param int[] $ignoreIds
      */
-    public function deleteAllTasks()
+    public function deleteAllTasks(array $ignoreIds = [])
```

Comment:
Should we maybe move this function to a trait, similar we do with `VersionDaoTrait`?

Reply:
See https://github.com/pimcore/pimcore/pull/10320/commits/305e744f98535945e427f478b96ed5e513f540a6